incubator/java-openliberty/templates/binary/src/main/liberty/config/server.xml

```diff
@@ -0,0 +1,11 @@
+<server description="Liberty server">
+    <featureManager>
+        <feature>microProfile-3.0</feature>
+    </featureManager>
+    <quickStartSecurity userName="admin" userPassword="adminpwd"/>
+    <keyStore id="defaultKeyStore" location="key.jks" type="jks" password="mpKeystore"/>
+    <httpEndpoint host="*" httpPort="${default.http.port}" 
+        httpsPort="${default.https.port}" id="defaultHttpEndpoint"/>
+    
+    <webApplication location="starter-app.war" contextRoot="/"/>
```

Comment:
Are we copying into this WAR name or using the original migrated app WAR name?

Reply:
we will preserve the original war name - the original war name will be the name of the app installed in the final image and will be the app name that should be reflected into any server.xml included in the project instance.

Reply:
so this app in the template server.xml isn't helping.  Maybe just add **src/main/liberty/config/.gitignore** to preserve the location but don't add server.xml?